scripts/gen-tx-gov.sh

```diff
@@ -0,0 +1,53 @@
+#! /bin/bash
+
+if test "$#" -ne 2
+then
+   echo "error: 2 parameters are expected (patch version and upgrade height number)"
+   echo "example : ./gen-tx-gov.sh v0.16.1 1336350"
+    exit
+fi
+
+if [[ ! "$1" =~ v[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3} ]]; 
```

Comment:
Create a separate var for the regex maybe? I would like to keep bash scripts as readable as possible so people not as familiar can maintain it when needed.

Reply:
good catch !